packages/strapi-helper-plugin/lib/src/components/HeaderSearch/index.js

```diff
@@ -6,16 +6,9 @@ import SearchInfo from '../SearchInfo';
 import Clear from './Clear';
 import Wrapper from './Wrapper';
 
-const HeaderSearch = ({
-  label,
-  name,
-  onChange,
-  onClear,
-  placeholder,
-  value,
-}) => {
+const HeaderSearch = ({ label, name, onChange, onClear, placeholder, value, className }) => {
   return (
-    <Wrapper>
+    <Wrapper className={className}>
```

Comment:
This is because I need to override the style of the headerSearch in the modal. You nee to add the className to a component to allow the styled-component override.